public/app/features/explore/Table/TableContainer.tsx

```diff
@@ -78,16 +79,16 @@ export class TableContainer extends PureComponent<Props> {
     return (
       <>
         {frames && frames.length === 0 && (
-          <PanelChrome title={'Table'} width={width} height={200}>
-            {() => <MetaInfoText metaItems={[{ value: '0 series returned' }]} />}
+          <PanelChrome title={t('table-container-title', 'Table')} width={width} height={200}>
+            {() => <MetaInfoText metaItems={[{ value: t('table-container-MetaInfoText', '0 series returned')}]} />}
```

Comment:
I think we should avoid mixing different capitalization styles and use lower case with dashes. Probably there's not need to specify the exact name of the component but use more contextual keys like, `explore.table.meta`. It'd be nice to also make use of [pluralization](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#plurals) here.

Reply:
Oh sorry, this message has a fixed count. Ignore my comment about pluralization.